src/loops/simple_programs_with_repetition/hard/q4/Reduceros.java

```diff
@@ -0,0 +1,61 @@
+package loops.simple_programs_with_repetition.hard.q4;
+
+/*
+Write a program to display a "single-digit reduction" of a number taken from the user by adding up its digits until
+a single-digit number is obtained. Eg. 567 becomes 5 + 6 + 7 = 18 = 1 + 8 => 9). Ensure your output matches the sample.
+
+Sample Output:
+Enter a number:
+567
+567's single number reduction is 9
+ */
+
```

Comment:
You should write a comment for this question that includes the reasoning behind the inclusion of this code. I know we talked about it in the pull request but if someone randomly stumbles upon this, I want it to make sense.